### PR TITLE
Fix TinyMCE error on product page

### LINKED ISSUE #20484: TinyMCE error on product page
**BO - Add product page (tinyMCE error)**
1. Go to 'BO -> products -> add product' page
2. Open console (`F12` in chrome then console tab)
3. See error
image

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | There was an error on the product page
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fix? | Fixes #20484
| How to test?  | Build assets (`cd admin-dev/themes/default && npm i && npm run build`)<br>Go on "create product" page, open the console, should have no errors related to tinyMCE (on Chrome), also the product page experience/work should be as fine as before, specially with tinyMCE fields

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20451)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

It is ok

**Before:**

image

**After**
image

But I found a difference comparing 1.7.7.x
**With PR**
Add a new product, click on the `Source code` of the Summary and Description => it is **empty**
image

**Without PR**, I have this
image

Also, for the demo products, there are differences
**With PR**
image

**Without PR**, I have this
image

Thanks!

**Comment by NeOMakinG** [comment]
Hey @khouloudbelguith,

I can't manage to reproduce it, maybe you added a space between branch switches ? I looks not related to what I touched on this PR :/

**Comment by matks** [comment]
Hi !

Maybe something was changed recently on `1.7.7.x` 🤔  because
- on 1.7.7.x I dont see the tinyMCE errors even though I dont checkout the branch for this PR
- on 1.7.7.x the "source code" for "summary" is empty even though I dont checkout the branch for this PR

@khouloudbelguith Can you please check again ?

**Comment by NeOMakinG** [comment]
@matks did you try when you'r on a "Create product" page ? The error in the issue was on this page

(Just changed the description for this, sorry :P )

**Comment by matks** [comment]
> @matks did you try when you'r on a "Create product" page ? The error in the issue was on this page
> 
> (Just changed the description for this, sorry :P )

Yes, on "Create a product". I see no error in Console.

Which is weird as I remember seeing them 😅

**Comment by NeOMakinG** [comment]
Also, did you build assets? (Trying to search why ^^)

**Comment by matks** [comment]
@matthieu-rolland does not see the issue neither.

There is something we are missing. Something related to our environments and how we run PrestaShop.

**Comment by NeOMakinG** [comment]
Found it, test it in low connection mode : 

image

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

> Hey @khouloudbelguith,
> 
> I can't manage to reproduce it, maybe you added a space between branch switches ? I looks not related to what I touched on this PR :/

I tried with a new shop => the same issue reproduced.
I'm using this command
`git prc origin 20451 1.7.7.x`

Thanks!

**Comment by NeOMakinG** [comment]
Hey @khouloudbelguith, thanks for testing

Which issue are you talking about ? The error in console, or the content of tinymce inputs ?

Thanks

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

This issue:
>But I found a difference comparing 1.7.7.x
**With PR**
Add a new product, click on the `Source code` of the Summary and Description => it is **empty**
image
**Without PR**, I have this
image
Also, for the demo products, there are differences
**With PR**
image
**Without PR**, I have this
image

Thanks!

**Comment by PierreRambaud** [comment]
> Hi @NeOMakinG,
> 
> This issue:
> 
> > But I found a difference comparing 1.7.7.x
> > **With PR**
> > Add a new product, click on the `Source code` of the Summary and Description => it is **empty**
> > image
> > **Without PR**, I have this
> > image
> > Also, for the demo products, there are differences
> > **With PR**
> > image
> > **Without PR**, I have this
> > image
> 
> Thanks!

@NeOMakinG @khouloudbelguith The div with display has nothing to do here since the it's not in the content. The value *With* the pull request is the good one. I also manage to reproduce the issue and everything is ok with this PR :)

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -79,10 +79,12 @@ $(document).ready(function() {
 function resetEditor() {
   const languageEditorsSelector = '.summary-description-container .panel.active div.translation-field.active textarea.autoload_rte';
   $(languageEditorsSelector).each(function(index, textarea) {
-    const editor = tinyMCE.get(textarea.id);
-    if (editor) {
-      //Reset content to force refresh of editor
-      editor.setContent(editor.getContent());
+    if(window.tinyMCE) {
```
Comment (PierreRambaud) [inline admin-dev/themes/default/js/bundle/product/form.js]:
```suggestion
    if (window.tinyMCE) {
```

**Review by PierreRambaud — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG, @PierreRambaud,

It is ok :heavy_check_mark: 
image

Thanks!

**Comment by NeOMakinG** [comment]
So @PierreRambaud, what about the strange div ? Do you have it locally ? I don't have it at all, but khouloud got it (I doubt this is linked to this PR, but let's be sure before merging)

**Comment by PierreRambaud** [comment]
> So @PierreRambaud, what about the strange div ? Do you have it locally ? I don't have it at all, but khouloud got it (I doubt this is linked to this PR, but let's be sure before merging)

Without the PR I don't have the empty div, it's maybe something added by a chrome extension :thinking:  Anyway, we can merge this one :)

**Comment by PierreRambaud** [comment]
Thank @NeOMakinG

**Pull request merged**